src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -221,8 +234,42 @@ const NodeDetails: React.FC<INodeProps> = ({ expandDrawer }) => {
             <Button onClick={downloadAllClick} icon={<FaDownload />}>
               Download Files
             </Button>
+            <Popover_patternfly
+                  id="node-details-popover"
+                  aria-label="Download command popover"
+                  position="bottom"
+                  headerContent={<div>chrs Download Files</div>}
+                  hasAutoWidth
+                  bodyContent={
+                    <div>
+                      <p>Copy and Paste the URL below into your chrs cli to download the files</p>
+                      <div className="pf-c-clipboard-copy">
+                      <div className="pf-c-clipboard-copy__group">
+                      <input className="pf-c-form-control" readOnly type="text" value={chrsCommandPath} aria-label="Readonly input example" />
```

Comment:
Prefer to use the React component from `@patternfly/react-core` instead of HTML elements + `className`.

https://www.patternfly.org/v4/components/clipboard-copy/#read-only

Reply:
Be more careful about leaving behind example code: `aria-label="Readonly input example"`